core/controllers/questions_list.py

```diff
@@ -24,6 +24,20 @@
 from core.domain import skill_domain
 from core.domain import skill_fetchers
 import feconf
+import utils
+
+
+def _require_valid_skill_ids(skill_ids):
+    """Checks whether the given skill ids are valid.
+
+    Args:
+        skill_ids: list(str). The skill ids to validate.
+    """
+    for skill_id in skill_ids:
+        try:
+            skill_domain.Skill.require_valid_skill_id(skill_id)
+        except Exception:
```

Comment:
except `ValidationError`.

Also, do we need try-except here, I don't think so!

Reply:
Yeah, not needed. 
Removed.